src/views/projectExplorer/memberFile.ts

```diff
@@ -14,48 +12,38 @@ import { ContextValue } from "../../projectExplorerApi";
  */
 export default class MemberFile extends ProjectExplorerTreeItem {
   static contextValue = ContextValue.memberFile;
```

Comment:
I am still not clear whata 'memberFile' is?

Reply:
MemberFile is a tree item for a member under a physical file.

Reply:
so this is the UI element?  The word "File" confused me - physical-file, stream file etc.  So would MemberItem be more clear?